### PR TITLE
Remove Conor and Pip from director list, adding Colin

### PR DESCRIPTION
**Pull request by andrewbolster**
Also tweaked about wording, added Colin, undated the "past officers" and a few other spring cleans

### CODE REVIEW

**Comment by chris18890** [comment]
killing approved :+1:

File: about/index.html [hunk 1]
```diff
@@ -96,6 +96,9 @@ <h5 class="subheader">{{officer.title}} ({{officer.year}} )</h5>
 			<li>Ben Bland: Director (2013-2014)</li>
 			<li>Rachel Gregg: Community NEM (2014-2015)</li>
 			<li>Matt Hazley: Projects NEM (2015)</li>
+			<li>Ben Bland: Development NEM (2014-2016)</li>
+      <li>Pip Shea: Director (2015-2016)</li>
```
Comment (chris18890) [inline about/index.html]: spacing not perfectly aligned </pedantic>

File: about/index.html [hunk 2]
```diff
@@ -96,6 +96,9 @@ <h5 class="subheader">{{officer.title}} ({{officer.year}} )</h5>
...
+      <li>Conor Robinson: Director (2014-2016)</li>
```
Comment (chris18890) [inline about/index.html]: as above

**Comment by Stealthii** [comment]
Renaming to actual, relevant title

File: about/index.html (hunk 1 above)
Comment (Stealthii) [inline about/index.html]: Spacing is pretty horrible as it stands, but keeping it consistent would be good.

**Comment by Stealthii** [comment]
Change to consistent (but broken) spacing for now.

**Comment by Stealthii** [comment]
Summarised changes properly:

* Move Conor and Pip from directors list to historical section
* Add Colin to directors list

**Pull request merged**